docs/src/pages/AndroidStarRatingInteractivePage/AndroidStarRatingInteractivePage.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import readme from '../../../../backpack-android/docs/InteractiveStarRating/README.md';
+import screenshotDefault from '../../../../backpack-android/docs/InteractiveStarRating/screenshots/default.png';
+import DocsPageBuilder from '../../components/DocsPageBuilder';
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default',
+    blurb:
+      'An interactive version of Star Rating. These can be set at half star intervals. ', // eslint-disable-line max-len
```

Comment:
Can we make the `Star rating` text link to the Star rating page?

Here's an example of how to do it!
https://github.com/Skyscanner/backpack-docs/blob/master/docs/src/pages/AlignmentPage/AlignmentPage.js#L69-L74